mockidentityprovider/src/api_endpoints/endpoint_root.js

```diff
@@ -7,6 +7,9 @@ async function getConfiguration (req, res) {
     token_endpoint: `${SERVER_URL}/oauth/token`,
     jwks_uri: `${SERVER_URL}/.well-known/jwks.json`
   }
+  // oas-tools Bug preventing using res.json
+  // https://github.com/oas-tools/oas-tools/issues/71
+  res.append('Content-Type', 'application/json;charset=utf-8')
```

Comment:
How was it working before then? Do we need to set the header because of updated npm packages? we are using `res.status(200).send` and not `res.json` anyway right?

Reply:
@k1ch Appending this header just resolved a warning in the logs that a `content-type` header was missing, so it defaulted to application/json anyway so there is no functional change. This just resolves the log file warning. 

If everything was working correct, then we could refactor the `res.status(200).send` to be just `res.json` and remove the explicit setting of the header as express would do this automatically by using `res.json`.